sdk/spring/scripts/replace_util.py

```diff
@@ -18,6 +18,13 @@
 VERSION_UPDATE_ITEMS = 'version_update_items'
 
 config = {
+    'appconfiguration': {
```

Comment:
Rather than this complexity, wouldn't it be better to just have two test modules with the different versions and use the POM to control where the sources are loaded from?

Reply:
> have two test modules

Test code can not be shared in 2 modules.
We just want to write test code in one  place.

Reply:
Remember that this complexity will make your build more brittle and harder for others to debug if something goes wrong. If there is a more "Maveny" way to do this with POM configuration then that would be better. Duplicating the tests might also be preferrable.

Reply:
> If there is a more "Maveny" way to do this with POM configuration then that would be better

I agree, but I didn't find that way.

> Duplicating the tests might also be preferrable.

If we have many tests, for example, 50 ~ 100 tests, change the pom file is better than keep same code in 2 place. We already discussed inside our team. And spring module and cosmos module already do like this.